src/helpers/DataRequirementHelpers.ts

```diff
@@ -511,3 +347,67 @@ export function getFlattenedRelatedArtifacts(
   // unique the relatedArtifacts
   return uniqBy(relatedArtifacts, JSON.stringify);
 }
+
+/**
+ *
+ * @param q The query which contains the filters to add to the data requirement
+ * @param dataRequirement Data requirement to add date filters to
+ * @param withErrors Errors object which will eventually be returned to the user if populated
+ * @returns void, but populated the dataRequirement filters
+ */
+export function addFiltersToDataRequirement(
```

Comment:
What's the difference between this function and the one that was renamed to `addGapFiltersToDataRequirement` in `GapsReportBuilder`?

Reply:
There is no difference. This is copied so that we can change things around on data requirements without interfering with gaps. Will DM with more info.